eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -19,10 +39,56 @@ steps:
         # Install gcc 9
         brew install gcc@9
         gcc --version
+    displayName: MacOS - Set tools versions
+    condition: >-
+      and(
+        succeeded(),
+        not(eq(variables['${{ parameters.DependenciesVariableName }}'], '')),
+        contains(variables['OSVmImage'], 'macOS')
+      )
+
+  # Clear the vcpkg folder so the cache can successfully deploy to that location
+  - bash: |
+      id
+      sudo rm -rf $(VCPKG_INSTALLATION_ROOT)
```

Comment:
Do we need to check if `VCPKG_INSTALLATION_ROOT` is set to a non-empty string?

Reply:
We can do that here but this is a Linux-specific step and that value is set on the Linux box.